projects/ion/src/lib/table/utilsTable.ts

```diff
@@ -44,6 +46,15 @@ interface LinkRow<RowType> {
   target?: LinkTarget;
   url?: (_: RowType) => string;
   action?: (_: RowType) => void;
+  tooltipConfig?: LinkTooltip<RowType>;
+}
+
+interface LinkTooltip<RowType> {
```

Comment:
You could extend from a Pick<T> or an Exclude<T>, where T is the IonTooltip component type.